addons/skin.estuary/xml/Includes_Buttons.xml

```diff
@@ -155,7 +155,7 @@
 				<enable>$PARAM[enable]</enable>
 			</control>
 		</definition>
-	</include>
+	</include> -->
```

Comment:
This was introduced by me during the Video Version work https://github.com/xbmc/xbmc/commit/cb42df816a1706df6e2865b952d00f91f502fa24
It was made redundant due to the many core refactors that allowed DialogSelect to be used, so I guess I forgot to go back and clean up after myself. Although everything was in such flux at that time perhaps I kept it aound in case it was needed. This can indeed be removed now.

Reply:
Ok, will remove and mark resolved.